internal/backends/qt/qt_widgets/listviewitem.rs

```diff
@@ -20,10 +20,27 @@ pub struct NativeStandardListViewItem {
     pub has_hover: Property<bool>,
     /// Specify that this item is in fact used in a ComboBox
     pub combobox: Property<bool>,
+    widget_ptr: std::cell::Cell<SlintTypeErasedWidgetPtr>,
+    animation_tracker: Property<i32>,
+}
+
+impl NativeStandardListViewItem {
+    /// Return the QWidget*
+    fn widget_ptr(&self) -> NonNull<()> {
+        let widget_ptr: *mut SlintTypeErasedWidgetPtr = self.widget_ptr.as_ptr();
+        cpp!(unsafe [widget_ptr as "std::unique_ptr<SlintTypeErasedWidget>*"] -> NonNull<()> as "void*" {
+            return (*widget_ptr)->qwidget();
+        })
```

Comment:
Idea: make this an associated method of `SlintTypeErasedWidgetPtr`

Reply:
Excellent idea!